examples/pytorch/rgcn-hetero/README.md

```diff
@@ -0,0 +1,54 @@
+# Relational-GCN
+
+* Paper: [https://arxiv.org/abs/1703.06103](https://arxiv.org/abs/1703.06103)
+* Author's code for entity classification: [https://github.com/tkipf/relational-gcn](https://github.com/tkipf/relational-gcn)
+* Author's code for link prediction: [https://github.com/MichSchli/RelationPrediction](https://github.com/MichSchli/RelationPrediction)
+
+The preprocessing is slightly different from the author's code. We directly load and preprocess
+raw RDF data. For AIFB, BGS and AM,
+all literal nodes are pruned from the graph. For AIFB, some training/testing nodes
+thus become orphan and are excluded from the training/testing set. The resulting graph
```

Comment:
I'm not sure if these isolated nodes should be removed. Even though they are isolated nodes, during BP implicitly their feature/label information also get incorporated into the weight matrices.

Reply:
There is no feature, so if the node has no connection, it simply trains a random embedding to fit the label. Anyhow, I will include the literal nodes so they are not isolated any more.

Reply:
Ok